projects/1.20-fabric/assets/paladins-and-priests/paladins/lang/zh_cn.json

```
@@ -0,0 +1,81 @@
+{
+  "itemGroup.paladins.general": "骑士与牧师",
+
+  "item.paladins.paladin_spell_book": "骑士圣契",
+  "item.paladins.priest_spell_book": "神圣书卷",
+
+  "item.paladins.paladin_armor_head": "骑士头盔",
+  "item.paladins.paladin_armor_chest": "骑士胸甲",
+  "item.paladins.paladin_armor_legs": "骑士护腿",
+  "item.paladins.paladin_armor_feet": "骑士靴子",
+  "item.paladins.crusader_armor_head": "十字军头盔",
+  "item.paladins.crusader_armor_chest": "十字军护甲",
+  "item.paladins.crusader_armor_legs": "十字军护腿",
+  "item.paladins.crusader_armor_feet": "十字军靴子",
+
+  "item.paladins.stone_claymore": "石双刃大剑",
+  "item.paladins.iron_claymore": "铁双刃大剑",
+  "item.paladins.golden_claymore": "金双刃大剑",
+  "item.paladins.diamond_claymore": "钻石双刃大剑",
+  "item.paladins.netherite_claymore": "下界合金双刃大剑",
+  "item.paladins.aeternium_claymore": "永恒双刃大剑",
```

貌似有一个阔剑的译名

---

`Claymore`确实有`苏格兰阔手大剑`的含义，刚刚搜索了一下剑桥是没收录这个词
但是《少年JUMP》有部漫画`クレイモア`大陆译名为`大剑`
此外，`Broadsword`同样译作`阔刀剑`
所以在此暂取`大剑含义`含义

---

```suggestion
  "item.paladins.aeternium_claymore": "太古合金双刃大剑",
```
疑似betterend兼容（不知道对不对）

---

草，等会看看betterend

---

还真是`太古合金锭`